src/main/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSink.java

```diff
@@ -263,6 +264,18 @@ public void configure(Context context) {
                     + "properly work!");
         } // if else
         
+        String defaultCsvSeparator = context.getString("default_csv_separator");
```

Comment:
You can simplify this code by doing the following:

    String csvSeparator = context.getString("csv_separator", ",");

That means "try getting from the configuration a parameter named 'csv_separator'; if not configured, the default CSV separator will the comma".